src/ParseContents.py

```diff
@@ -8,7 +8,7 @@ def helperRemoveTrailingChars(s):
     if s[0] == ".":
         s = s[1:]
 
-    if s[-1] == ".":
+    if len(s) > 1 and s[-1] == ".":
```

Comment:
Added length check, the function crashed the parser when the word "content" was followed by some string similar in format to the ones used in actual content of the parser